Gemfile

```diff
@@ -26,6 +27,8 @@ gem 'paranoia'
 gem 'pg'
 gem 'puma'
 gem 'pundit'
+gem 'rails-i18n', '~> 5.0.0'
```

Comment:
The gemspec file will ensure, that bundler takes the right version: https://github.com/svenfuchs/rails-i18n/blob/master/rails-i18n.gemspec#L22